build-tools/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
-install(PROGRAMS
+#
+# CASA build-tools
+#
+
+install (PROGRAMS
   casacore_assay
   casacore_floatcheck
+  casacore_memcheck
 DESTINATION bin
 )
+
+install (FILES
+  casacore_memcheck.supp
+DESTINATION share
```

Comment:
Julian,
Thanks. You're right. I'll change it right away.
Ger

On Tue, Jun 23, 2015 at 3:19 PM, Julian Taylor <notifications@github.com>
wrote:

> In build-tools/CMakeLists.txt
> <https://github.com/casacore/casacore/pull/130#discussion_r33039090>:
>
> >  DESTINATION bin
> >  )
> > +
> > +install (FILES
> > +  casacore_memcheck.supp
> > +DESTINATION share
>
> shouldn't this be share/casacore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/casacore/casacore/pull/130/files#r33039090>.
>